crates/server-fixture/server/Cargo.toml

```diff
@@ -14,9 +14,14 @@ hyper-util = { workspace = true, features = ["full"] }
 tokio = { workspace = true, features = ["macros", "rt-multi-thread"] }
 tokio-util = { workspace = true, features = ["compat", "io"] }
 tower-service = { version = "0.3" }
-
+serde_json = { workspace = true }
 tlsn-server-fixture-certs = { workspace = true }
 
 [[bin]]
-name = "main"
+name = "tlsn-server-fixture"
 path = "src/main.rs"
+
+[dev-dependencies]
+axum-test = "16.2.0"
```

Comment:
nit: i think we're adopting the format of always using `{ version = "16.2.0" }` in the project?

Reply:
fixed